.circleci/config.yml

```diff
@@ -0,0 +1,34 @@
+version: 2
+jobs:
+  integration:
+    machine:
+      image: circleci/classic:latest
+    steps:
+      - checkout
+      - run: "./.circleci/init-env.sh > tests/env.sh"
+      - run: "./integrationTest.sh"
+  unit:
+    docker:
+      - image: circleci/golang:1.11
+    steps:
+      - checkout
+      - run: go test ./...
+  lint:
+    docker:
+      - image: circleci/golang:1.11
+    steps:
+      - checkout
+      - run: "! go fmt ./... 2>&1 | read"
```

Comment:
This trick comes from here: https://circleci.com/blog/enforce-build-standards/

Basically it is just checking whether the command outputs anything. If the command has any output, it failed.